bundle/manifests/splunk-operator.clusterserviceversion.yaml

```diff
@@ -802,14 +802,14 @@ spec:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: RELATED_IMAGE_SPLUNK_ENTERPRISE
-                  value: splunk/splunk:9.0
+                  value: splunk/splunk:9.1
```

Comment:
I think, this needs to be a different image with the fix for kvstore issue:
CC: @gaurav-splunk